python/lsst/ip/isr/calibType.py

```diff
@@ -261,11 +270,44 @@ def updateMetadata(self, camera=None, detector=None, filterName=None,
         self._metadata["DET_SER"] = self._detectorSerial if self._detectorSerial else None
         self._metadata["FILTER"] = self._filter if self._filter else None
         self._metadata["CALIB_ID"] = self._calibId if self._calibId else None
+        self._metadata["SEQFILE"] = self._seqfile if self._seqfile else None
+        self._metadata["SEQNAME"] = self._seqname if self._seqname else None
+        self._metadata["SEQCKSUM"] = self._seqcksum if self._seqcksum else None
```

Comment:
How come these are hard-coded still, even though this is now a config option? Doesn't this now need to be done dynamically somehow from the list in the config?

Reply:
Reply in next comment.